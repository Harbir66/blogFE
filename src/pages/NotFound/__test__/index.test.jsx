import { render } from '@testing-library/react';
import React from 'react';
import PageNotFound from '..';

describe('Not Found Page', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<PageNotFound />);
    expect(asFragment()).toMatchSnapshot();
  });
});
