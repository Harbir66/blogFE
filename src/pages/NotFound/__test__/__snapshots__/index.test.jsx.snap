// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found Page should render correctly 1`] = `
<DocumentFragment>
  <div
    class="main"
  >
    <header>
      <div
        class="padding"
      >
        <h1>
          The Artifact
        </h1>
        <p>
          <em>
            Culture & Art blog
          </em>
        </p>
      </div>
      <nav
        class="padding"
      >
        <ul>
          <li>
            <a
              href="./"
            >
              Blog
            </a>
          </li>
          <li>
            <a
              href="./"
            >
              About
            </a>
          </li>
          <li>
            <a
              href="./"
            >
              Contact
            </a>
          </li>
        </ul>
      </nav>
    </header>
    <div
      class="pageNotFoundContainer"
    >
      <p>
        404 Error. Page not found
      </p>
    </div>
    <footer
      class="padding"
    >
      <p>
        Â©artifact.com 2019
      </p>
    </footer>
  </div>
</DocumentFragment>
`;
